projects/dot-rules/src/lib/condition-types/serverside-condition/serverside-condition.ts

```diff
@@ -21,19 +21,17 @@ import { LoggerService } from 'dotcms-js';
       <cw-input-dropdown *ngIf="input.type == 'dropdown'"
                          flex
                          class="cw-input"
-                         [hidden]="input.argIndex !== null && input.argIndex >= _rhArgCount"
-                         [formControl]="input.control"
-                         [required]="input.required"
+                         (touch)="onBlur(input)"
                          [allowAdditions]="input.allowAdditions"
                          [class.cw-comparator-selector]="input.name == 'comparison'"
                          [class.cw-last]="islast"
-                         (touch)="onBlur(input)"
-                         placeholder="{{input.placeholder | async}}">
-        <cw-input-option
-            *ngFor="let opt of input.options"
-            [value]="opt.value"
-            [label]="opt.label | async"
-            icon="{{opt.icon}}"></cw-input-option>
+                         [formControl]="input.control"
+                         [hidden]="input.argIndex !== null && input.argIndex >= _rhArgCount"
```

Comment:
`_rhArgCount`  better/clearer variable name?

Reply:
This was a prettier fix, there are a lot of bad naming vars in here. Not part of this issue to rename it, BUT (big but) I think we're in a place we can refactor rules and make it better and keep working on it.